src/scripts/towns/NPCController.ts

```diff
@@ -1,9 +1,17 @@
 class NPCController {
     public static selectedNPC: KnockoutObservable<NPC> = ko.observable();
-
+    private static modalState: any;
     public static openDialog(npc: NPC) {
         this.selectedNPC(npc);
         $('#npc-modal').modal();
         npc.setTalkedTo();
+        if (!this.modalState) {
+            this.modalState = DisplayObservables.modalState['npc-modalObservable'].subscribe((value: BootstrapState) => {
+                if (value === 'hide') {
```

Comment:
```suggestion
                if (value === 'hidden') {
```
hide - as the modal begins closing
hidden - after it's fully closed

prevents the model content from disappearing as it closes